public/plugin.tsx

```diff
@@ -100,6 +102,24 @@ export class CustomImportMapPlugin
       },
     });
 
+    core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [{
+      id: MAPS_APP_ID,
+      category: DEFAULT_APP_CATEGORIES.visualizeAndReport,
+      order: 200,
+    }]);
+
+    core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS['security-analytics'], [{
+      id: MAPS_APP_ID,
+      category: DEFAULT_APP_CATEGORIES.visualizeAndReport,
+      order: 200,
+    }]);
+
+    core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.search, [{
+      id: MAPS_APP_ID,
+      category: DEFAULT_APP_CATEGORIES.visualizeAndReport,
```

Comment:
```suggestion
      category: DEFAULT_APP_CATEGORIES.analyzeSearch,
```
Seems the category should be DEFAULT_APP_CATEGORIES.analyzeSearch for search use case.

Reply:
Updated